.circleci/config.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           command: sleep 500
       - run:
           name: Check 404s production
-          command: yarn blc https://allcontributors.org -ro || true
+          command: npx blc https://allcontributors.org -ro || true
```

Comment:
This dependency hasn't been updated in 4 years. we should consider running lychee here via a github action and remove this. i opened #852